pxtblocks/blocklyimporter.ts

```diff
@@ -2,6 +2,10 @@
 /// <reference path="../built/pxtlib.d.ts" />
 
 namespace pxt.blocks {
+    export interface BlockSnippet {
```

Comment:
add the "versions" field of targets so that down the road we can do clever app compat to import snippets.